packages/web-extension/test/walletManager/walletManagerWorker.test.ts

```diff
@@ -41,48 +54,74 @@ const expectWalletChannelClosed = () => {
 
 describe('WalletManagerWorker', () => {
```

Comment:
We should probably add a couple new tests for new functionality

Reply:
Once we decide the actual interface functionality of the wallet manager, I will then proceed to complete the unit tests as the last step

Reply:
added some more tests to cover the extra cases here: https://github.com/input-output-hk/cardano-js-sdk/pull/1012/commits/ee775430185f2444c530835a5ce0306e00a4986c